spec/base.spec.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+define(['ably', 'globals'], function(Ably, ablyGlobals) {
+  describe('Static generated ably.js library', function() {
+    it('is loaded into the global namespace', function() {
+      expect(Ably).not.toBeUndefined();
+    });
+  });
+
+  describe('Globals in support library', function() {
+    var isBrowser = (typeof(window) == 'object');
+    var environment = isBrowser ? window.__env__ : process.env;
+
+    it('contains a valid environment', function() {
+      expect(ablyGlobals.environment).toEqual(environment.ABLY_ENV || 'sandbox');
```

Comment:
Either you expect ABLY_ENV, either you expect `'sandbox'`, but you can't have an `||` in this expectation. It's confusing.